clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/BaseIntegrationSpec.scala

```diff
@@ -80,6 +89,81 @@ abstract class BaseIntegrationSpec(clioDescription: String)
     */
   def elasticsearchUri: Uri
 
+  /** URL of vault server to use when getting bearer tokens for service accounts. */
+  private val vaultUrl = "https://clotho.broadinstitute.org:8200/"
+
+  /** Path in vault to the service account JSON to use in testing. */
+  private val vaultPath = "secret/dsde/gotc/test/clio/clio-account.json"
+
+  /** List of possible token-file locations, in order of preference. */
+  private val vaultTokenFiles = Seq(
+    new File("/etc/vault-token-dsde"),
```

Comment:
I think in general we should prefer `Path` over `File`, since `Path` is the newer API.